CHANGELOG.md

```diff
@@ -21,6 +19,11 @@ Nothing yet.
   - hue rotate
   - invert
   - saturate
+- Removed redundant dependencies that were being unnecessarily installed by users using npm to install ulabel
+- Updated webpack build process to properly provide both a minified (default) and unminified build (for better debugging)
+  - Added package `exports` field with options: `.` (minified), `./min` (minified), `./debug` (unminified)
```

Comment:
This does have the downside of adding a massive diff to every PR. Not a huge deal, although I would lean on the side of having all of the builds done as a GH Action upon merging PRs into main (which would also automate away one step of the current PR checklist).